crates/turborepo-lib/src/run/cache.rs

```diff
@@ -463,3 +471,10 @@ impl ConfigCache {
         Ok(FileHashes(hash_object).hash())
     }
 }
+
+// attempt to write message to writer, swallowing any errors encountered
+fn failable_write(mut writer: impl Write, message: &str) {
```

Comment:
This is the existing behavior of `PrefixedUI::output`

Reply:
Small nit, I think rust tends to use fallible over failable